content/tracing/faq/traces-sampling-and-storage.md

```diff
@@ -0,0 +1,112 @@
+
+* **Client sampling**: Allows reduction of instrumentation overhead by only instrumenting a configurable percentage of the transactions. By default, it is disabled.
+
+{{< img src="tracing/faq/sampling_trace.png" alt="Individual traces are sampled at the agent, server, and client level." responsive="true" popup="true" style="width:90%;">}}
+
+### Signature Sampling
+
+Signature Sampling ensures a sampling of a variety of [traces](/tracing/services/trace) (errors, successes) for each [resource](/tracing/services/resource) (endpoint, database query).  
+
+Datadog computes a *signature* for every trace reported, based on its services, resources, errors, etc.. Traces of the same signature are considered similar. For example, a signature could be:
+
+* `env=prod`, `my_web_service`, `is_error=true`, `resource=/login`
+* `env=staging`, `my_database_service`, `is_error=false`, `query=SELECT…``
+
+A proportion of traces with each signature is then kept, so you get full visibility into all the different kinds of transactions happening in your system. This method ensures traces for resources with low volumes are still kept.  
+
+Both the [Datadog Agent](/agent) and backend apply Signature Sampling to limit network consumption and the total volume of stored traces while still making a representative set of traces available to you.
+
+### Priority Sampling for Distributed Tracing 
+
+Because Signature Sampling decisions are made at the [Agent](/agent) level, it is not guaranteed that a trace will be complete when one is running a distributed architecture and requests are across multiple services, hosts, containers etc... This is because each host would need to choose to sample (keep) spans from the same trace.  
+
+**Priority Sampling is an additional sampling option in which you indicate whether a trace should be kept. Priority Sampling runs prior to Signature Sampling, and a decision is made at the beginning of a trace**. As the decision is propagated in the trace context, it has two main properties:
+
+* It ensures completeness of traces distributed across hosts and services
+* Important traces can be kept
+
+#### How it works
+
+Each trace has a `sampling_priority`, which is assigned at its inception and propagated along the entire trace:
+
+| Value      |  Type | Action|
+| :--------------------- | :---------------- | :----------|
+|**-1**|User input| The Agent drops the trace. |
+|**0**|Automatic sampling decision| The Agent drops the trace. |
+|**1**|Automatic sampling decision| The Agent drops the trace. |
+|**2**|User input| The Agent keeps the trace, and the backend will only apply sampling if above maximum volume allowed. |
+
+Traces are automatically assigned a priority of **0** or **1**, with a proportion ensuring that the Agent won't have to sample more than it is allowed.  Override it by assigning a value of **2** if it is a trace that should be kept (critical transaction, debug mode, etc.).  
```

Comment:
"The user can override this by assigning a value of **2**..."

Reply:
👍